quantus_sdk/lib/src/services/datetime_formatting_service.dart

```diff
@@ -23,6 +42,35 @@ class DatetimeFormattingService {
     return '${_formatPast(difference)} $timeString';
   }
 
+  static FormattedDuration formatDuration(Duration? duration) {
+    if (duration == null || duration.isNegative) {
```

Comment:
Are we ever expecting to pass in a null duration here? 

Seems we should crash when this happens, rather than show 00:00:00